assets/src/components/activities/DeliveryElement.ts

```diff
@@ -299,6 +301,10 @@ export abstract class DeliveryElement<T extends ActivityModelSchema> extends HTM
     const mode = valueOr(this.getAttribute('mode'), 'delivery') as DeliveryMode;
     const sectionSlug = valueOr(this.getAttribute('section_slug'), undefined);
     const userId = this.getAttribute('user_id') as any;
+    let bibParams = valueOr(this.getAttribute('bib_params'), undefined);
+    if (bibParams) {
+      bibParams = JSON.parse(atob(bibParams));
```

Comment:
IMO you should always wrap `JSON.parse` in a try/catch as it can be exception prone